vaadin-renderer-flow-parent/vaadin-renderer-flow/src/main/resources/META-INF/resources/frontend/lit-renderer.ts

```diff
@@ -37,28 +37,24 @@ _window.Vaadin.setLitRenderer = (
 ) => {
   // Dynamically created function that renders the templateExpression
   // inside the given root element using Lit
-  const renderFunction = Function(`
-    "use strict";
 
-    const [render, html, live, returnChannel] = arguments;
-
-    return (root, model, itemKey) => {
```

Comment:
These arguments are no longer available from the template expression. Especially not having `model` available is problematic, the tests in `TreeComponentColumnsIT` are failing due to this.

Reply:
Are those arguments part of the API contract? (Though I guess we might have to preserve them regardless for backward compatibility reasons...)

Reply:
[`model` is at least](https://vaadin.com/docs/latest/components/grid/renderers#accessing-model-properties). I'm not sure if anyone's using `root` or `itemKey` for anything but maybe it's safer to include them also

Reply:
Added arguments, hopefully it's ok now. `TreeComponentColumnsIT` seems to pass now at least

Reply:
Looks good. `html, live, itemKey, model, item, index` would be a nicer order for the args but not a blocker for merging.